wiki/History_of_osu!/2015/en.md

```diff
@@ -1,39 +1,185 @@
 
 ## July
 
-peppy and flyte briefly explained why the temporary osu!web redesign subdomain was `jizz.ppy.sh` in an episode of osu!talk.
+On 15 July 2015, it was announced that a new public [Slack](https://slack.com/ "Slack") team was created for the purpose of better communicating with public contributors. [peppy](https://osu.ppy.sh/users/2) initially made the announcement on his blog and had explained that anyone who was willing to help test cutting-edge features, contribute graphical/sound resources, or get help working with osu! services could come and join the team and talk to the osu! team directly [[21]][r].
+
+*Note: There was still a private Slack team that the osu! team used for internal communications, which was not accessible or viewable to outside users; these two teams were independent of each other.*
+
+Notwithstanding, the new team was completely public and available for anyone to join, which caused many unwanted users to flood the channels. This forced peppy into "locking down" the new team the very next day. Despite this, peppy continued to make use of the team throughout the rest of the year, and often invited users to come and join when they were testing new features or implementations [[22,23]][r].
+
+## August
+
+The Chat Moderators were officially merged into the Global Moderation Team (GMT) on 20 August 2015 [[24,25]][r].
+
+## September
+
+The old FPS counter was replaced with a new "refresh and latency display" on 2 September 2015. This new display would still display the current FPS (frames per second), but the new change caused it to hide from view if it exceeded 500 FPS. In addition, a new counter was added below the FPS counter that would display exactly how much latency was being introduced during frame rendering (measured in milliseconds). "Stuttered" or "dropped" frames would be shown as coloured rectangles to the left of the counter to better visualise exactly how much frame render latency a user was experiencing [[26,27]][r].
+
+On 26 September 2015, [cYsmix](http://cysmix.com/ "cysmix.com") became one of the first official osu! artists. This partnership entailed the creation of four tracks made specifically for osu!, along with the creation of a mapping contest using those tracks. The aforementioned four tracks were as follows:
+
+- ["Manic"](https://soundcloud.com/olemlanglie/cysmix-manic-osu)
+- ["House with Legs"](https://soundcloud.com/olemlanglie/cysmix-house-with-legs-osu)
+- ["Dovregubben's Hall (New Ver.)"](https://soundcloud.com/olemlanglie/cysmix-dovregubbens-hall-new-ver-osu)
+
+![](img/cysmix-album-cover-1.jpg "The album cover for cYsmix's first release of official osu! songs")
+
+Each track had its own pre-timed [`.osz`](/wiki/osu!_File_Formats/Osz_(file_format)) file included with it and were released on [cYsmix's SoundCloud](https://soundcloud.com/olemlanglie "SoundCloud") on 28 September 2015.
```

Comment:
```suggestion
Each track had its own pre-timed [`.osz`](/wiki/osu!_File_Formats/Osz_(file_format)) file included with it. They were released on [cYsmix's SoundCloud](https://soundcloud.com/olemlanglie "SoundCloud") on 28 September 2015.
```

Reply:
one sentence sounds a little better imo; will unresolve if you disagree